boards/luatos/esp32s3_luatos_core/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ESP32S3_LUATOS_CORE
+	select SOC_ESP32_PROCPU if BOARD_ESP32S3_LUATOS_CORE_ESP32S3_PROCPU
+	select SOC_ESP32_APPCPU if BOARD_ESP32S3_LUATOS_CORE_ESP32S3_APPCPU
+
+config BOARD_ESP32S3_LUATOS_CORE_USB
+	select SOC_ESP32_PROCPU if BOARD_ESP32S3_LUATOS_CORE_USB_ESP32S3_PROCPU
+	select SOC_ESP32_APPCPU if BOARD_ESP32S3_LUATOS_CORE_USB_ESP32S3_APPCPU
```

Comment:
This causes a problem because for the opposite board it will not be defined, 2 possible solutions:

1. Make the usb a variant of the main board, then you can use the base board name here once instead
2. Keep boards differently then use something like this:
```
config BOARD_ESP32S3_LUATOS_CORE_EXTRA_CPU
	bool
	default y
	depends on BOARD_ESP32S3_LUATOS_CORE || BOARD_ESP32S3_LUATOS_CORE_USB
	select SOC_ESP32_PROCPU if BOARD_ESP32S3_LUATOS_CORE_ESP32S3_PROCPU || BOARD_ESP32S3_LUATOS_CORE_USB_ESP32S3_PROCPU
	select SOC_ESP32_APPCPU if BOARD_ESP32S3_LUATOS_CORE_ESP32S3_APPCPU || BOARD_ESP32S3_LUATOS_CORE_USB_ESP32S3_APPCPU
```